blocks/large-manual-promo-block/features/large-manual-promo/default.jsx

```diff
@@ -50,6 +167,18 @@ LargeManualPromo.propTypes = {
 			defaultValue: false,
 			group: "Configure Content",
 		}),
+		author: PropTypes.string.tag({
```

Comment:
We should not be adding additional content as part of this work - that is out of scope and not part of this deliverable

Reply:
Removed it.